input-output-hk/lace
# feat(staking): LW-8148 match multistaking overview empty state with designs

# Checklist

- [x] JIRA - LW-8148

---

## Proposed solution

Restructure the elements and add styles to make the empty state screen match the Figma designs (see JIRA task for link)

## Testing

1. set up a wallet with no staking and non-empty funds
2. go to "Staking" section in the expanded view
3. check that the "Get started" section matches the figma designs in both light and dark mode

## Screenshots

Before:

<img width="903" alt="Screenshot 2023-08-29 at 02 43 11" src="https://github.com/input-output-hk/lace/assets/4980147/8b79716d-f53e-439d-96d5-07bd227f13bc">

After:

light mode:

<img width="912" alt="Screenshot 2023-08-29 at 02 44 00" src="https://github.com/input-output-hk/lace/assets/4980147/79476a3b-6aec-4044-b3d1-782cbfb9e34e">

dark mode:

<img width="780" alt="Screenshot 2023-08-29 at 02 44 16" src="https://github.com/input-output-hk/lace/assets/4980147/f67c9516-4483-43b5-a59c-96f176e79645">



---
# Allure report
`allure-report-publisher` generated test report!



**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1782/6023588460/index.html) for [cb2b3898](https://github.com/input-output-hk/lace/pull/457/commits/cb2b3898abaa5829ad1d4446c0395ecec9c4d933)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 34     | 1      | 0       | 0     | 35    | ❌     |
